kytos.json

```diff
@@ -3,7 +3,7 @@
   "username": "kytos",
   "name": "topology",
   "description": "Manage the network topology.",
-  "version": "2023.1.0",
+  "version": "2023.1.1",
```

Comment:
Looks good, but in regarding the target branch if you could target `base/2023.1.1` instead. You can branch it from `2023.1.0` and push it upstream. This is a similar suggestion that I made on mef_eline, but I haven't pushed `base/2023.1.1` upstream on topology yet, feel free to do it. If you need help let me know. In summary, at the moment, we're always creating a new `base/x.y.z` branch before adding a new tag.

Reply:
Created `base/2023.1.1` branch